src/db/pool.cr

```diff
@@ -37,8 +37,15 @@ module DB
     # global pool mutex
     @mutex : Mutex
 
-    def initialize(@initial_pool_size = 1, @max_pool_size = 0, @max_idle_pool_size = 1, @checkout_timeout = 5.0,
-                   @retry_attempts = 1, @retry_delay = 0.2, &@factory : -> T)
+    @connection_ttl : Int32
```

Comment:
can either safely remove this or need to add `@reaper_retry` -- I think the former